editor/src/clj/editor/library.clj

```diff
@@ -99,17 +99,25 @@
     (io/copy is f)
     f))
 
+(defn- replace-user-info-env-variable [^String user-info-token]
+  (if (and (some? user-info-token)
+           (str/starts-with? user-info-token "__")
+           (str/ends-with? user-info-token "__"))
+    (let [env-key (subs user-info-token 2 (- (count user-info-token) 2))
+          env-value (System/getenv env-key)]
+      (or env-value user-info-token))
+    user-info-token))
+
 (defn- make-basic-auth-headers
   [^String user-info]
-  (let [user-info-parts (str/split user-info #":")
-        username (get user-info-parts 0)
-        password (or (get user-info-parts 1) "")]
-    (if (and (str/starts-with? password "__") (str/ends-with? password "__"))
-      (let [password-env-key (subs password 2 (- (count password) 2))
-            password-env-value (or (System/getenv password-env-key) password)]
-         {"Authorization" (format "Basic %s" (str->b64 (str username ":" password-env-value)))})
-      {"Authorization" (format "Basic %s" (str->b64 user-info))})))
-  
+  (let [[username password] (str/split user-info #":")
+        username (replace-user-info-env-variable username)
+        password (replace-user-info-env-variable password)
+        user-info (cond-> username
+                          password (str ":" password))
+        user-info-b64 (str->b64 user-info)
+        authorization (format "Basic %s" user-info-b64)]
+    {"Authorization" authorization}))
```

Comment:
Previously only the password part of the URL supported environment variable replacement, but according to the docs we should also support it for the username: https://defold.com/manuals/libraries/#basic-access-authentication